docs/docusaurus/versioned_docs/version-0.18/cloud/about_gx.md

```diff
@@ -1,26 +1,20 @@
 ---
-sidebar_label: 'About GX Cloud'
-title: 'About GX Cloud'
-description: Learn more about the features and benefits of GX Cloud.
+sidebar_label: 'GX Cloud deployment patterns and architecture'
+title: 'GX Cloud deployment patterns and architecture'
+description: Learn more about GX Cloud deployment patterns and architecture.
 toc_min_heading_level: 2
 toc_max_heading_level: 2
 ---
 
-GX Cloud builds on the innovative features of GX open source software (OSS). Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
+GX Cloud simplifies data quality management and monitoring. Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud, you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
```

Comment:
GX Cloud simplifies data quality management and monitoring. Its easy-to-use web interface lets you quickly validate your data without provisioning and maintaining complex infrastructure. With GX Cloud, you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.